bin/auto-cpufreq

```diff
@@ -41,7 +42,7 @@ def main(monitor, live, daemon, install, log, debug):
                     sysinfo()
                     set_autofreq()
                     countdown(5)
-                    run("clear")
```

Comment:
Why was this function call removed?

Reply:
`clear_logs` automatically clears the console or logfile depending on which is used. If the log file is used it runs `truncate(0)` which clears the file, otherwise it does `run("clear")`.

Reply:
Okay, now I see it, good point and thanks for clarification.